packages/@aws-cdk/aws-logs/test/log-retention.test.ts

```diff
@@ -200,6 +176,137 @@ describe('log retention', () => {
     });
   });
 
+  test('multiple log retention resources with removalPolicy DESTROY', () => {
```

Comment:
can we break this up into two tests in the following format:

```ts
describe('multiple log retention resources', () => {
  test('both removalPolicy DESTROY', () => {
  });
  test('with removalPolicy DESTROY and removalPolicy RETAIN', () => {
  });
});
```

hopefully that make sense. just trying to split these up into units and have descriptive test names.